### PR TITLE
correct typing for ``removeAllListeners``

### PR DESCRIPTION
**Pull request by gitawego**

### CODE REVIEW

**Comment by gitawego** [comment]
this could fix the issue #199

File: eventemitter2.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
 declare let eventemitter2: eventemitter2.Static
 
 declare module eventemitter2 {
+    type evetns = string[];
```
Comment (RangerMauve) [inline eventemitter2.d.ts]: You misspelled "events" as "evetns". Mind fixing that?

Comment (gitawego) [inline eventemitter2.d.ts]: I think the origial author wanted to call it: event namespace, for short, it's ``evetns`` :)
if you think it's not very clear, I can rename it

Comment (RangerMauve) [inline eventemitter2.d.ts]: Hmm, I think that "events" might be better. Or at the very least "eventns"

Comment (gitawego) [inline eventemitter2.d.ts]: it's not for events, but for event namespaces in this function IMO. so I think ``eventns`` or ``eventNS`` might be better :)

Comment (RangerMauve) [inline eventemitter2.d.ts]: Would you mind changing it to eventNS, then?

**Comment by RangerMauve** [comment]
Thank you very much for the PR! It's released as part of 2.2.2

**Pull request merged**